assets/scss/coder.scss

```diff
@@ -1,3 +1,4 @@
+@import "css/normalize", "css/fork-awesome";
```

Comment:
Perhaps there is a SCSS version of these dependencies. But it is ok to keep it as CSS files for now.

Reply:
no scss versions, but with this it merges them to main css file during build which essentially reduces loading time = less css files = less http(s) requests